src/sentry/models/groupsnooze.py

```diff
@@ -88,13 +91,7 @@ def is_valid(self, group=None, test_rates=False, use_pending_data=False):
             if self.user_window:
                 if not self.test_user_rates():
```

Comment:
Ahh I was confused, I guess we also need caching like this here, but that could be a follow up